public-api/src/main/resources/public-api.yaml

```diff
@@ -359,6 +359,10 @@ definitions:
         description: prevalence among participants percent count divided num participants
         type: number
         format: float
+      subQuestionCount:
```

Comment:
This shouldn't be necessary anymore, can just use length of `subQuestions` right?

Reply:
Removed it.